packages/core/src/internal-components/close-button/close-button.element.scss

```diff
@@ -29,6 +31,7 @@
   transform: translateY(#{$cds-token-space-size-1});
 }
 
-:host() cds-icon {
-  --color: var(--color);
+cds-icon {
+  --color: var(--icon-color);
+  cursor: pointer;
```

Comment:
For some reason the cursor is ignored unless explicitly on the icon in chrome. The specificity problem I found was due to this line `--color: var(--color);` as a var assignment will refer to its own scope instead of the one on the `:host`. The way to use the same name on the host and child is to have an intermediate var on `.private-host` that can pass it down.